08-annotation.Rmd

```diff
@@ -75,11 +75,11 @@ install.packages("pheatmap")
 
 ```{r, warning = FALSE, message = FALSE}
 # AnVIL::install("pheatmap")
-#library(pheatmap)
+library(pheatmap)
```

Comment:
do we need to add this to the docker image?

Reply:
It would probably be nice.

Reply:
Though the code actually works without it (and with the local installation of pheatmap)